docs/sources/write/style-guide/word-list/index.md

```diff
@@ -30,7 +30,7 @@ This page is a work in progress.
 
 **data source** - Use this rather than _datasource_ for the noun form.
 
-When you need to use the adjective form, use _data-source_ rather than _datasource_.
+Also, use _data source plugin_ rather than _data-source plugin_.
```

Comment:
```suggestion
Also, use _data source plugin_ rather than _data-source plugin_.

{{% admonition type="note" %}}
For other compound adjectives, use a hyphen unless otherwise specified.
{{% /admonition %}}
```